pkg/diagnostics/networkpod/util/util.go

```diff
@@ -31,6 +31,8 @@ const (
 	NetworkDiagNodeLogDirPrefix      = "/nodes"
 	NetworkDiagMasterLogDirPrefix    = "/master"
 	NetworkDiagPodLogDirPrefix       = "/pods"
+
+	NetworkDiagDefaultPodImage = "openshift/origin"
```

Comment:
Actually, it does need to use the defaults correctly since enterprise would then fail to pull diagnostics.  It can be a follow up issue but would need to be fixed for 1.5